src/main/java/com/azure/cosmos/examples/patch/sync/SamplePatchQuickstart.java

```diff
@@ -0,0 +1,490 @@
+        cosmosPatchOperations.add("/vaccinated", false).replace("/district", "new_replaced_value");
+
+        CosmosPatchItemRequestOptions options = new CosmosPatchItemRequestOptions();
+
+        logger.info("predicate " + predicate);
+        options.setFilterPredicate(predicate);
+
+        // predicate match failure will result in BadRequestException
+        try {
+            CosmosItemResponse<Family> response = this.container.patchItem(id, new PartitionKey(partitionKey),
+                    cosmosPatchOperations, options, Family.class);
+            logger.info("Item with ID " + response.getItem().getId() + " has been patched");
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void patchIncrement(String id, String partitionKey) {
+        logger.info("Executing Patch with 'increment' operations");
+
+        CosmosPatchOperations cosmosPatchOperations = CosmosPatchOperations.create();
+        cosmosPatchOperations
+                // attribute does not exist, will be added
+                .add("/num", 42)
+                // attribute already exists, will be replaced
+                .increment("/num", 1)
+                // negative increment works is equivalent to decrement
+                .increment("/num", -1);
+
+        CosmosPatchItemRequestOptions options = new CosmosPatchItemRequestOptions();
+        try {
+            CosmosItemResponse<Family> response = this.container.patchItem(id, new PartitionKey(partitionKey),
+                    cosmosPatchOperations, options, Family.class);
+            logger.info("Item with ID " + response.getItem().getId() + " has been patched");
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    // demonstrates how to remove an attribute, including array
+    private void patchRemove(String id, String partitionKey) {
+        logger.info("Executing Patch with 'remove' operations");
+
+        CosmosPatchOperations cosmosPatchOperations = CosmosPatchOperations.create();
+
+        cosmosPatchOperations
+                // removes this attribute
+                .remove("/registered")
+                // removes the element from an array
+                .remove("/children/0/pets/0");
+
+        // un-commenting this will cause an exception since the attribute does not exist
+
+        // cosmosPatchOperations.remove("/attribute_does_not_exist");
+
+        CosmosPatchItemRequestOptions options = new CosmosPatchItemRequestOptions();
+        try {
+            CosmosItemResponse<Family> response = this.container.patchItem(id, new PartitionKey(partitionKey),
+                    cosmosPatchOperations, options, Family.class);
+            logger.info("Item with ID " + response.getItem().getId() + " has been patched");
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void patchTransactionalBatch() {
```

Comment:
can we bundle multiple patch operations with multiple request of even multiple document and also add one conditional case in of of the request. This will give a overall variation view in batch request.

Reply:
sure, makes sense